spec/features/admin_can_login_to_admin_dashboard_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.